gatsby-theme-landing-page/src/components/divider.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import * as styles from "./divider.module.css";
+
+const dividerSize = {
+  S: styles.short,
+  M: styles.half,
+  half: styles.half,
+  L: styles.long,
+  full: styles.full,
+};
+
+export default function Divider({ size }) {
+  const widthStyle = dividerSize[size] || styles.half;
+  return <div className={`${styles.divider} ${widthStyle}`} />;
```

Comment:
Any reason not to use `<hr>` here?

Reply:
I honestly think its just easier to reason about styling behavior for `divs` vs `hr` (https://stackoverflow.com/questions/6382023/changing-the-color-of-an-hr-element).